atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ValidatingQueryRewritingKeyValueService.java

```diff
@@ -103,7 +103,7 @@ protected static void sanityCheckTableName(TableReference tableRef) {
                 "invalid tableName: " + tableName);
     }
 
-    protected static void sanityCheckTableMetadata(TableReference tableRef, byte[] tableMetadata) {
+    protected void sanityCheckTableMetadata(TableReference tableRef, byte[] tableMetadata) {
```

Comment:
I made it non-static in case the internal product has a need to override it. but can switch it back as its not needed for this PR.